Add changelog and fix unpacking bug

sampaccoud opened the pull request:
## Purpose

I noticed that one of our long standing and intensive bug was fixed in openedx master.

### Proposal

- [x] Port the fix to our new `dogwood.3-fun` branch
- [x] Add a CHANGELOG.md file to start tracking changes

jmaupetit commented on CHANGELOG.md:
```diff
@@ -0,0 +1,29 @@
+- Correctly use read_preference setting in doc_store_config
+
+[unreleased]: https://github.com/openfun/edx-platform/compare/v5.3.0...dogwood.3-fun
+[5.2.0] https://github.com/openfun/edx-platform/compare/fun-5.2.0...v5.3.0
```
```suggestion
[5.3.0] https://github.com/openfun/edx-platform/compare/fun-5.2.0...v5.3.0
```

sampaccoud commented:
@jmaupetit I fixed the mistake you pointed out and replaced my fix by a cherry-pick of the original commit https://github.com/edx/edx-platform/commit/cab6c9fcbd9e19ba3d89b80ab901087dfbf3e64f
